src/js/store/make.js

```diff
@@ -36,7 +36,7 @@ export default function(Vuex, plugins) {
       },
       loadLanguage(state, {key, data}) {
         state.languages[key] = data
-      }
+      },
```

Comment:
I had been playing around with this and the thought of storing my data model in vuex.  It never really got working properly (without a lot more effort), so I abandoned it.  I left these trailing commas probably on purpose.  We can discuss the style implications, but it basically boils down to this.  Say you want to add a new mutation after loadLanguage, you might add the new function, but forget to add the comma at the end of the loadLanguage function.  I make this mistake often enough, and I sometimes waste time finding the error.  The better reason to do this though is for the sake of versioning.  Take the same scenario, when you do a code review the diff will mark not only the new function we added, but also the last line of the loadLanguage function (because we added a comma).  It is not too confusing in this case, but can waste some time when you don't immediately realize the difference is only the presence of a comma.  Not a big deal, let me know your preference and I am happy to conform.

Reply:
One comment here re: styles that came up: I honestly don't care that much (2 spaces vs 4), etc. I care about 2 things: consistency within the codebase and clarity (code can be easily scanned to see what it is doing). Style only aid that. So, I'm happy to say that we standardize around trailing commas if it makes your work easier ... but I'm adding a ticket to implement prettier so we get automatic consistency on all this stuff.

Reply:
Cool, that will be very nice!